ondemand.osc.edu/central_config.yaml

```diff
@@ -0,0 +1,261 @@
+OOD_APP_CONFIG_ROOT: null
+## OOD_APP_DEVELOPMENT
+# Purpose: Flag whether the developer view should be shown in the Dashboard
+# Note: Considers whether the development app directory exists in the configured path, or is a symlink; see also OOD_DEV_APPS_ROOT
+# Type: Boolean | null
+# Default: false
+OOD_APP_DEVELOPMENT: null
+## OOD_APP_SHARING
+# Purpose: Flag whether the app sharing controls should be shown in the Dashboard
+# Type: Boolean | null
+# Default: false
+OOD_APP_SHARING: false
+## OOD_BC_APP_CONFIG_ROOT
+# Purpose: The file system path to the default configuration files for Batch Connect applications
+# Type: String | null
+# Default: /etc/ood/config/apps
+OOD_BC_APP_CONFIG_ROOT: null
+## OOD_BRAND_BG_COLOR
+# Purpose: Color for the navbar's background (CSS colors)
+# Type: String | null
+# Default: #53565a (Grey)
+OOD_BRAND_BG_COLOR: '#6CACE4'
+## OOD_BRAND_LINK_ACTIVE_BG_COLOR
+# Purpose: Color for the navbar links' background when clicked (CSS colors)
+# Type: String | null
+# Default: #3b3d3f (Darker grey)
+OOD_BRAND_LINK_ACTIVE_BG_COLOR: '#375C84'
+## OOD_DASHBOARD_2FA_URL
+# Purpose: ???
+# TODO: What is the purpose of this? If a user can see this link then they have already authenticated. If they need help with auth, then they won't be able to see this link. ???
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_2FA_URL: null
+## OOD_DASHBOARD_DEV_DOCS_URL
+# Purpose: An external link to documentation on developing applications for OnDemand
+# Type: String | null
+# Default: https://go.osu.edu/ood-app-dev
+OOD_DASHBOARD_DEV_DOCS_URL: null
+## OOD_DASHBOARD_DOCS_URL
+# Purpose: An external link to documentation for OnDemand users
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_DOCS_URL: https://www.osc.edu/ondemand
+## OOD_DASHBOARD_PASSWD_URL
+# Purpose: An external link to your site's password changing service
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_PASSWD_URL: https://my.osc.edu
+## OOD_DASHBOARD_SUPPORT_URL
+# Purpose: An external link to your site's user support
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_SUPPORT_URL: https://www.osc.edu/contact/supercomputing_support
+## OOD_DASHBOARD_HEADER_IMG_LOGO
+# Purpose: ???
+# Type: String | null
+# Default:
+OOD_DASHBOARD_HEADER_IMG_LOGO: null
+## OOD_DASHBOARD_LOGO
+# Purpose: A link to an image to use in the Dashboard's header
+# Replaces: DISABLE_DASHBOARD_LOGO
+# Example: /public/logo.png
+# Type: String | null
+# Default: null
+OOD_DASHBOARD_LOGO: /public/logo.png
+## OOD_DASHBOARD_SUPPORT_EMAIL
+# Purpose: The email address for your site's user support staff
+# Type: String | null
+# Default:
+OOD_DASHBOARD_SUPPORT_EMAIL: oschelp@osc.edu
+## OOD_DASHBOARD_TITLE
+# Purpose: The title for your OnDemand instance; displayed on the Dashboard
+# Type: String | null
+# Default: Open OnDemand
+OOD_DASHBOARD_TITLE: OSC OnDemand
+## OOD_DATAROOT
+# TODO: revisit this
+# Purpose: ???
+# Type: String | null
+# Default:
+OOD_DATAROOT: null
```

Comment:
The problem with dataroot here, is that the dataroot of each app is different. There may be a common dataroot template string that might look like `%{userhome}/%{portal}/%{apptoken}`. But then you have apps like the Job Composer where you can change both the database path and the dataroot - so these can be in two separate locations.

Reply:
Removed.